### ISSUE TITLE
Incompatible with ejected create-react-app

### ISSUE THREAD

**Issue by behnammodi** [johanholmerin/style9#39]
I tried to use style9 in ejected CRA but it not works.

How reproduce:

## Install CRA
`npx create-react-app s9 --use-npm`

## Eject
`npm run eject`

## Add style9 config to
`./config/webpack.config.js`

## Add smaple code to `App.js` ex:
```jsx
import style9 from 'style9';

const styles = style9.create({
  blue: {
    color: 'blue',
  },
  red: {
    color: 'red'
  }
});

function App() {
  return (
    <h1 className={styles('blue', true && 'red')>Hi Style9</h1>
  );
}

export default App;

```

## Run
`npm run start`

**Comment by johanholmerin** [comment]
Fixed in 0.10.2

**Comment by behnammodi** [comment]
@johanholmerin Thanks 😂 you're very fast
